data/json/npcs/NC_HUNTER.json

```diff
@@ -67,23 +79,65 @@
     "id": "NC_HUNTER_misc",
     "items": [
       [ "sinew", 10 ],
-      [ "water_clean", 20 ],
-      [ "meat_cooked", 10 ],
+      [ "bolt_wood", 90 ],
+      [ "bolt_wood", 90 ],
```

Comment:
You should use syntax like `{ "item": "bolt_wood", "prob": 90, "count": 2 }`. Here and below.

Reply:
Corrected